brewtils/decorators.py

```diff
@@ -276,7 +276,11 @@ def echo(self, message):
 
 
 def parameters(*args, **kwargs):
-    """Decorator for specifying multiple Parameter definitions at once
+    """
+    .. deprecated:: 3.0
+        Will be removed in version 4.0. Please use ``@command`` instead.
+
+    Decorator for specifying multiple Parameter definitions at once
```

Comment:
I think technically you are correct that the google style guide dictates that the docstring should start with the summary. I had it that way originally and ended up switching it because:

* I liked the final output of the sphinx docs better that way.
* Sphinx didn't seem to mind, meaning it didn't throw any warnings or anything at me.

If someone feels super strongly about it I don't mind changing it, but otherwise I'd value the slight improvement to the rendered output over strict adherence to the google style.

Reply:
This seems good to me, I think it's good to put the focus on the deprecation.